scripts/quickstart.py

```diff
@@ -324,7 +344,7 @@ def create_dsr_policy(key: str):
     if response.ok:
         policies = (response.json())["succeeded"]
         if len(policies) > 0:
-            logger.info("Created fidesops policy with key=%s via {url}", key)
+            logger.info("Created fidesops policy with key={} via {}", key, url)
```

Comment:
This had been causing the quickstart to break FYI, not sure if people are using the quickstart

Reply:
I believe @seanpreston said it was still being used for manual testing. If that's the case I think we should also be running it during CI checks (another ticket for another time)